MAINTAINERS.yml

```diff
@@ -0,0 +1,1009 @@
+#
+#     files:
+#         List of paths and/or glob patterns giving the files in the area,
+#         relative to the root directory.
+#
+#         If a path or glob pattern ends in a '/', it matches all files within
+#         the given directory or directories. Otherwise, an exact match is
+#         required.
+#
+#         Paths to directories should always have a trailing '/'.
+#
+#     files-regex:
+#         List of regular expressions applied to paths to determine if they
+#         belong to the area. The regular expression may match anywhere within
+#         the path, but can be anchored with ^ and $ as usual.
+#
+#         Can be combined with a 'files' key.
+#
+#         Note: Prefer plain 'files' patterns where possible. get_maintainer.py
+#         will check that they match some file, but won't check regexes
+#         (because it might be slow).
+#
+#     files-exclude:
+#         Like 'files', but any matching files will be excluded from the area.
+#
+#     files-regex-exclude:
+#         Like 'files-regex', but any matching files will be excluded from the
+#         area.
+#
+#     description: >-
+#         Plain-English description. Describe what the system is about, from an
+#         outsider's perspective.
+#
+#
+# All areas must have a 'status' key and a 'files' and/or 'files-regex' key.
+# The other keys are optional.
+#
+#
+# Workflow
+# ########
+#
+# Ideally, any file in the tree will be covered by some area.
+#
+# When a GitHub pull request is sent, this happens:
+#
+#     * Users mentioned in 'maintainers' and 'collaborators' are added as
+#       reviewers to the pull request
+#
+#     * The labels listed in 'labels' are automatically added to the pull
+#       request
+#
+#     * The bot posts this comment:
+#
+#         This PR affects the following areas:
+#         <area name>:
+#           Status: ...
+#           Maintainers: <list of maintainers>
+#           Collaborators: <list of sub-maintainers>
+#           CC: <list of informed GitHub users, like @johndoe>
+#
+#         <area name>:
+#           ...
+#
+#     * An e-mail is sent to the addresses in 'inform'
+#
+# Changes to MAINTAINERS.yml need to be approved as follows:
+#
+#     * Changing the 'maintainers' for an area needs approval from the
+#       technical steering committee
+#
+#     * Changing the 'collaborators' lines requires the maintainer and
+#       collaborators of that area to agree (or vote on it)
+
+# Areas are sorted by name
+
+"ARC arch":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - ruuddw
+    collaborators:
+        - vonhust
+    files:
+        - arch/arc/
+        - include/arch/arc/
+    labels:
+        - "area: ARC"
+
+"ARM arch":
+    # Provisional entry, to be corrected
+    status: maintained
+    maintainers:
+        - ioannisg
+    collaborators:
+        - MaureenHelm
+        - galak
+        - ioannisg
```

Comment:
not sure if it makes sense for a person to be both collaborator and maintainer

Reply:
Nope, that would be a mistake

Reply:
Good. There exist some additional candidates, here. I suppose they can be added any time later, with approval from Maureen and Kumar.

Reply:
Yep, I would think it would be better to do that after this is merged after you say, have the maintainers update their entries after discussing with collaborators. Doing it now could make this PR quite long :)